script/Arcueid/Day 7/07_00_ARC07_2_1.txt

```diff
@@ -12,8 +12,9 @@ Surely she's calmed down by now, right?"
 -- Page 160, Offset 16970.
 -- 秋葉が怒鳴った理由は、俺が廊下を走った事と、診察を受けなかった事だ。
 // 診察を受け -> Take his temperature, since that's what she was trying to do in that scene
+// It sounds like Akiha is the subject of the second clause
 She'd yelled at me earlier because I'd been running in the hallway,
- and had also refused to let her take my temperature.
+ and I had also refused to let her take my temperature.
```

Comment:
I can see how it can be read that way, but the repeated I had feels a bit stiff. Maybe
> The two things she'd yelled at me for earlier were running in the hallways and refusing to let her take my temperature.